sdk/common/src/main/java/io/opentelemetry/sdk/internal/InstantClock.java

```diff
@@ -6,29 +6,31 @@
 package io.opentelemetry.sdk.internal;
 
 import io.opentelemetry.sdk.common.Clock;
+import java.time.Instant;
 import java.util.concurrent.TimeUnit;
 import javax.annotation.concurrent.ThreadSafe;
 
 /** A {@link Clock} that uses {@link System#currentTimeMillis()} and {@link System#nanoTime()}. */
 @ThreadSafe
-public final class MillisClock implements Clock {
+public final class InstantClock implements Clock {
```

Comment:
To avoid renaming this class in the future whenever the implementation changes (this is a public class!), I strongly suggest choosing a more neutral name like SystemClock or UnixTimeClock.